transform/utils.go

```diff
@@ -84,6 +85,8 @@ func convertBoolean(raw interface{}) (interface{}, error) {
 		return t > 0, nil
 	case nil:
 		return nil, nil
+	case []*xmlquery.Node:
+		return strconv.ParseBool(t[0].InnerText())
```

Comment:
I think this should happen before calling the utils, ie the `t[0].InnerText()` part. At the least if it stays this way you will need to expand the unit tests for conversion.